src/main/java/org/folio/rest/impl/HoldingsStorageAPI.java

```diff
@@ -408,4 +431,60 @@ private boolean isUUID(String id) {
       return false;
     }
   }
+
+  private CompletableFuture<Void> updateItemEffectiveCallNumbersByHoldings(AsyncResult<SQLConnection> connection, PostgresClient postgresClient, HoldingsRecord holdingsRecord) {
+    CompletableFuture<Void> future = new CompletableFuture<>();
```

Comment:
Maybe use a Vert.x `Future` or, better yet, `Promise` here and elsewhere instead of introducing `CompletableFuture`. It appears all we need this for is indicating a PostgresClient operation is complete. Instead of `thenAccept`, we could use `setHandler`, since a Promise is already a handler, it can be passed directly to the `setHandler` method to complete it.

For example:
```java
  final Promise<Void> promise = Promise.promise();
  // stuff
  updateEffectiveCallNumbers(/*args*/).setHandler(promise);
  // stuff
  return promise.future(); // or just "return promise;"
```